lmd/objects.go

```diff
@@ -203,6 +203,8 @@ func NewContactsTable() (t *Table) {
 	t = &Table{DefaultSort: []string{"name"}}
 	t.AddColumn("alias", Static, StringCol, "The full name of the contact")
 	t.AddColumn("can_submit_commands", Static, IntCol, "Wether the contact is allowed to submit commands (0/1)")
+	t.AddColumn("custom_variable_names", Static, StringListCol, "A list of all custom variables of the contact")
+	t.AddColumn("custom_variable_values", Dynamic, StringListCol, "A list of the values of all custom variables of the contact")
```

Comment:
@sni : What is supposed to happen if this column is set to `Static` instead?

I tried changing it and couldn't observe anything different in behaviour. WIth debug logging enabled, I saw a line like this:
```
[2020-08-19 15:03:27.218][Debug][peer.go:1435] [Local Livestatus] fetched table:        contacts - time:       0s - count:        3 - size:        0 kB
```
once at start and then after every time contacts were changed. This seems to be the same thing no matter if this column is Dynamic or Static.